test/main_test.cpp

```diff
@@ -0,0 +1,18 @@
+#include <gtest/gtest.h>
+
+#include "rclcpp/node.hpp"
```

Comment:
Hier sollte [rclcpp/utilities.hpp](https://docs.ros.org/en/iron/p/rclcpp/generated/file_include_rclcpp_utilities.hpp.html#file-include-rclcpp-utilities-hpp) reichen.
```suggestion
#include "rclcpp/utilities.hpp"
```

Reply:
Hat so auch funktioniert, hab ich aber jetzt geändert.